puzzle-utils/words/dropquote.py

```diff
@@ -17,6 +17,13 @@ def __init__(self, index, word, letters, row, column, possibilities):
         self.row = row
         self.column = column
         self.possibilities = possibilities
+    
+    def print(self):
```

Comment:
The pythonic way to do this is use `__str__` that returns the string representation of the object